lib/config/validation.ts

```diff
@@ -182,7 +183,7 @@ export async function validateConfig(
               !(val && val.length === 1 && val[0] === '*')
             ) {
               try {
-                RegExp(val as any);
+                regEx(val as any);
                 if (!safe(val as any)) {
```

Comment:
We can remove the `safe` check here and instead catch the `config-validation` error that would be thrown